### PR TITLE
add member

### PR DESCRIPTION
**Pull request by NguyenHai2310**

### CODE REVIEW

File: app/views/admin/team_users/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<% provide :title, t "team.tmember" %>
+<h1>Assign User</h1>
```
Comment (vjnhat) [inline app/views/admin/team_users/show.html.erb]: i18n

File: app/views/admin/team_users/_form_user.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<%= form_for [:admin, @team] do |f| %>
+  <%= render "shared/error_messages", object: f.object %>
+    <table class="table">
+      <tr>
+        <th><%= t "team.user_in_team" %><th>
+      </tr>
+      <%#= byebug %>
```
Comment (vjnhat) [inline app/views/admin/team_users/_form_user.html.erb]: remove this line

File: app/controllers/admin/teams_controller.rb [hunk 3]
```diff
@@ -3,10 +3,12 @@ class Admin::TeamsController < ApplicationController
 
   def show
     @team = Team.find params[:id]
+    @leader = User.find @team.user_id
   end
 
   def new
     @team = Team.new
+    @users = User.leader
```
Comment (khanhkhanh) [inline app/controllers/admin/teams_controller.rb]: Change @users => @leaders to make it clearer

File: app/controllers/admin/teams_controller.rb [hunk 4]
```diff
@@ -46,6 +52,7 @@ def destroy
 
   private
   def team_params
-    params.require(:team).permit :name, :description
+    params.require(:team)
+      .permit :name, :description, :user_id, user_ids: []
```
Comment (khanhkhanh) [inline app/controllers/admin/teams_controller.rb]: 2 params: user_id and user_ids???

File: app/views/admin/teams/_form.html.erb [hunk 5]
```diff
@@ -6,7 +6,11 @@
       <%= f.text_field :name, class: "form-control" %>
       <%= f.label :description %>
       <%= f.text_field :description, class: "form-control" %></br>
+      <div class="field">
+        <%= f.label :user_id, "Leader" %>
```
Comment (phonghenry14) [inline app/views/admin/teams/_form.html.erb]: i18n

File: app/views/admin/team_users/show.html.erb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<% provide :title, t "team.tmember" %>
```
Comment (phonghenry14) [inline app/views/admin/team_users/show.html.erb]: chỗ này ko bỏ dc ()

File: app/views/admin/teams/index.html.erb [hunk 7]
```diff
@@ -1,5 +1,8 @@
 <% provide :title, t("team.index.title") %>
 <h1><%= t "team.index.title" %></h1>
+<%= link_to root_path do %>
```
Comment (phonghenry14) [inline app/views/admin/teams/index.html.erb]: sao lại phải viết do end ở chỗ này?

File: app/views/layouts/application.html.erb [hunk 8]
```diff
@@ -16,4 +16,4 @@
       <%= render 'layouts/footer' %>
```
Comment (phonghenry14) [inline app/views/layouts/application.html.erb]: " "

File: app/controllers/admin/teams_controller.rb [hunk 9]
```diff
@@ -3,10 +3,12 @@ class Admin::TeamsController < ApplicationController
 
   def show
     @team = Team.find params[:id]
+    @leader = User.find @team.user_id
   end
 
   def new
     @team = Team.new
+    @users = User.select_leader
```
Comment (khanhkhanh) [inline app/controllers/admin/teams_controller.rb]: Use @leaders to make it clearer

File: app/controllers/admin/teams_controller.rb [hunk 10]
```diff
@@ -3,10 +3,12 @@ class Admin::TeamsController < ApplicationController
 
   def show
     @team = Team.find params[:id]
+    @leader = User.find @team.user_id
```
Comment (khanhkhanh) [inline app/controllers/admin/teams_controller.rb]: @leader = User.find @team.user_id => Incorrect
??? Just use @leader = @team.leader ???

File: app/controllers/admin/teams_controller.rb [hunk 11]
```diff
@@ -21,16 +23,20 @@ def create
 
   def edit
     @team = Team.find params[:id]
+    @users = User.select_leader
```
Comment (khanhkhanh) [inline app/controllers/admin/teams_controller.rb]: Use @leaders to make it clearer

File: app/views/admin/team_users/show.html.erb (hunk 6 above)
Comment (khanhkhanh) [inline app/views/admin/team_users/show.html.erb]: You should check after changing something

File: app/views/admin/teams/index.html.erb (hunk 7 above)
Comment (khanhkhanh) [inline app/views/admin/teams/index.html.erb]: do - end is correct here.
Look carefully and you can see he is using glyphicon (Bootsrap).
And this is a new thing to you, Phong :)

File: app/controllers/admin/teams_controller.rb [hunk 12]
```diff
@@ -3,17 +3,19 @@ class Admin::TeamsController < ApplicationController
 
   def show
     @team = Team.find params[:id]
+    @leader = User.find @team.leader_id
   end
 
   def new
     @team = Team.new
+    @leader = User.select_leader
```
Comment (khanhkhanh) [inline app/controllers/admin/teams_controller.rb]: @leaders, not @leader

File: app/controllers/admin/teams_controller.rb [hunk 13]
```diff
@@ -21,16 +23,20 @@ def create
 
   def edit
     @team = Team.find params[:id]
+    @leader = User.select_leader
```
Comment (khanhkhanh) [inline app/controllers/admin/teams_controller.rb]: @leaders, not @leader

**Pull request merged**